libs/openchallenges/home/src/lib/home.component.html

```diff
@@ -1,4 +1,21 @@
 <main class="base mat-typography">
+  <div id="announcement">
+    <h4>
+      Welcome to
+      <a href="/about" target="_blank">OpenChallenges</a>!
+    </h4>
+    <h3>This website uses a preview of the OpenChallenges (OC) Database</h3>
+    <h4>
+      We are working hard to update the DB with more challenge data. The comprehensive DB will be
+      released in September or October 2023. We encourage organizations to
```

Comment:
Actually let's say September 2023 to make the message sound more definite. That gives us 6 weeks after Sage Week, which should be enough.

```suggestion
      released in September 2023. We encourage organizations to
```